src/Annotation.php

```diff
@@ -26,7 +26,9 @@ public static function create(array $data): self
         $instance->id = $data['id'];
         $instance->image_id = $data['image_id'];
         $instance->category_id = $data['category_id'];
-        $instance->segmentation = is_array($data['segmentation'][0]) ? $data['segmentation'][0] : $data['segmentation'];
+        $instance->segmentation = isset($data['segmentation']) 
+            ? (is_array($data['segmentation'][0]) ? $data['segmentation'][0] : $data['segmentation'])
+            : null;
```

Comment:
I suggest that you set `segmentation = []` if no data is provided then you don't have to update all the "isPointShape" etc. methods below. Just check if there is a bbox first and if not, use the previous logic for the segmentation.